src/server/pfs/cmds/cmds.go

```diff
@@ -658,6 +662,9 @@ want to consider using commit IDs directly.
 			var path string
 			if len(args) == 3 {
 				path = args[2]
+				if _, err := url.Parse(path); err == nil {
+					fmt.Printf("warning: path \"%s\" looks like a URL; did you mean -f <url>?\n", path)
```

Comment:
A a warning I like it, seems like a good ergonomic improvement.
2 minor things

- Should this be going to `stderr` along with the one below? And probably the "reading from stdin" message as well.
- What if we said "did you mean to put from it with: `-f %s` and spliced in their url? That gives them an easy route to understanding the concept.